be/src/pipeline/pipeline_fragment_context.cpp

```diff
@@ -512,12 +515,28 @@ Status PipelineFragmentContext::submit() {
     if (_submitted) {
```

Comment:
In which case this `submit` method will be called multiple times?

Reply:
At present, there is no logic to sbumit twice.